app/src/main/res/layout/feedback_item.xml

```diff
@@ -68,6 +48,16 @@
       android:text="@{htmlContent}"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="16sp"
-      android:visibility="@{htmlContent.length() > 0 ? View.VISIBLE : View.GONE, default=gone}" />
```

Comment:
`FeedbackItem` can have `internal-links` which are clickable and therefore the min-height should be `48dp`. Now consider a case where the text is only of `1 line`, in that case the min-height is less than `48dp` and therefore I have shifted the padding of `FrameLayout` to `TextView` and also I have increased the padding-top/bottom from 12dp to 16dp in `dimens.xml` files too.

Same concept applies for `ContentItem` too.